definitions/json/CaseEvent/CaseEvent-ET1Repped.json

```diff
@@ -11,7 +11,10 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Create draft claim",
-    "ShowSummary": "N"
+    "ShowSummary": "N",
+    "SignificantEvent": "Yes",
+    "TTLIncrement": "365",
+    "Publish": "Y"
```

Comment:
It might be better to break this event into a prod and non prod version. Prod version being the event before, non-prod being the version with TTL changes. Also don't need the publish